sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -292,6 +292,27 @@ private[sql] object QueryParsingErrors extends QueryErrorsBase {
       ctx)
   }
 
+  def nestedTypeMissingElementTypeError(
+      dataType: String, ctx: PrimitiveDataTypeContext): Throwable = {
+    dataType match {
+      case "array" =>
+        new ParseException(
+          errorClass = "INCOMPLETE_TYPE_DEFINITION.ARRAY",
+          messageParameters = Map("elementType" -> "<elementType>"),
```

Comment:
why do we need the parameter here? Isn't it always hard code?

Reply:
@gengliangwang Because of a limitation of the current framework replacing all `<([a-zA-Z0-9_-]+)>` as a parameter and throws an exception if it's undefined.

If I do `For example: Array<elementType>` as is, it throws an undefined parameter exception.

I wish to keep `<` `>` without using a parameter, do you have better ways to keep `<` `>` as is?